src/freenas/usr/local/share/ses_firmware/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+  "file": "rainbow_4.3.0.5_CombinedImage.bin",
+  "version": "0430"
```

Comment:
@darkfiberiru is `rainbow_4.3.0.6_CombinedImage.bin` possible? What will be a string in `camcontrol devlist` then?